src/cli/init.ts

```diff
@@ -49,10 +54,16 @@ export default class InitCommand extends BaseCommand {
     if (!isAddress(offerId)) throw new Error('Invalid Offer Address')
 
     if (fs.existsSync(this.dbPath as string)) {
-      if (!this.parsedArgs.flags.skipPrompt && !(await this.confirm('Are you sure you want to overwrite your current DB? All data will be erased! (y/n)'))) {
+      const overrideDb = this.parsedArgs.flags['override-db']
+
+      if (overrideDb === false) {
         this.exit()
-      } else {
+      }
+
+      if (overrideDb === true || this.parsedArgs.flags.skipPrompt === true || await this.confirm('Are you sure you want to overwrite your current DB? All data will be erased! (y/n)')) {
         fs.unlinkSync(this.dbPath as string)
+      } else {
+        this.exit()
```

Comment:
I don't know if this translates into a process `exit`. If that's the case, it will be great to exit with an error code, to signal that the process aborted because it isn't allowed to overwrite the DB.

Reply:
Since no value is specified in the call, it exits with exit code 0, which is correct because it is not an error exit, but just "early termination".